src/agent/v8debugapi.ts

```diff
@@ -208,7 +215,9 @@ export function create(
         logger.info('deactivating v8 breakpoint listener');
         v8.setListener(null);
       }
-      return true;
+      return setImmediate(function() {
```

Comment:
No need to change this but for future reference this can be written: `setImmediate(cb, null)`.